StepDefinitions/LanguageTestStepDefinitions.cs

```diff
@@ -0,0 +1,91 @@
+using MarsQA.Helpers;
+using MarsQA.Pages;
+
+namespace MarsSpecFlow.StepDefinitions
+{
+    [Binding]
+    public class LanguageTestStepDefinitions : Driver
+    {
+        LanguagePage languagePage = new LanguagePage();
+        
+        [Given(@"I create a new langauge records with values  '([^']*)' '([^']*)'")]
+        public void ThenICreateANewLangaugeRecordsWithValues(string language, string level)
+        {
+            languagePage.SuccessfullyAddNewLanguage(driver, language, level);
```

Comment:
Please do not pass the driver In action methods instead get the pages to extend to common driver where you will create a common driver class which will initialise the web driver 

public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }

Reply:
please change this every where

Reply:
I have code change to inherit  driver class from the pages and remove from action method